angular-mentoring/src/app/main/main.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="main__search">
```

Comment:
It would be better to extract this search input to the separate component

Reply:
I separated search block into the component.